packages/manager/apps/dedicated/client/app/ip/components/list/firewall/game/ip-ip-firewall-game.html

```diff
@@ -194,23 +194,61 @@
                 <tr
                     data-ng-repeat="rule in IpGameFirewallCtrl.rules | orderBy : '-id'"
                 >
-                    <th
-                        scope="row"
-                        data-ng-if="!rule.errorMessage"
-                        data-ng-bind="rule.protocol"
-                    ></th>
-                    <td
-                        data-ng-if="!rule.errorMessage"
-                        data-ng-bind="rule.ports.from"
-                    ></td>
-                    <td
-                        data-ng-if="!rule.errorMessage"
-                        data-ng-bind="rule.ports.to"
-                    ></td>
+                    <td scope="row" data-ng-if="!rule.errorMessage">
+                        <select
+                            required
+                            id="protocol"
+                            name="protocol"
+                            class="form-control"
+                            data-ng-model="IpGameFirewallCtrl.editRule.protocol"
+                            data-ng-options="IpGameFirewallCtrl.getProtocoleText(protocol) for protocol in IpGameFirewallCtrl.enums.protocols"
+                            data-ng-if="rule.isEditable"
```

Comment:
this condition could be set on the parent td, this way you wouldn't have to set it on both child input